scripts/taco_to_burrito.py

```diff
@@ -30,33 +34,51 @@ def main():
     for file in os.scandir(source_path):
         if(file.name.endswith(".xml")):
             full_path: str = os.path.join(source_path, file.name)
+            error_in_new_file = True
```

Comment:
Instead of having several uncorrelated global flags that must be set in order for eprint to work properly just have a single setter that sets all of the values at once so future users are less likely to accidentally break something in a non obvious way..